benchmark-service/benchmark/benchmark.py

```diff
@@ -195,7 +197,7 @@ async def index(request):
     d = {
         'dates': benchmark_data['dates'],
         'geo_means': benchmark_data['geo_means'],
-        'commits': benchmark_data['commit_ids']
+        'commits': benchmark_data['pr_ids']
```

Comment:
Let's rename this something else. This is confusing with the commits that means something else in benchmark_data.